src/SignalR/clients/java/signalr/src/main/java/com/microsoft/signalr/HubConnection.java

```diff
@@ -521,6 +525,7 @@ private void stopConnection(String errorMessage) {
             handshakeResponseSubject.onComplete();
             redirectAccessTokenProvider = null;
             connectionId = null;
+            connectionToken = null;
```

Comment:
Are we trying to follow any convention related to using or not using `this.` to access fields?

Reply:
We should, but not currently. It's just random right now.